### PR TITLE
feat(icon): export `bell-slash` and `bell-snooze` icon

### PR DESCRIPTION
**Pull request by musingh1**
image

image

image

image

### CODE REVIEW

File: packages/react/src/themes/teams/components/Icon/svg/icons/index.ts [hunk 1]
```diff
@@ -166,6 +168,7 @@ export default {
   'arrow-right': arrowRight,
   'audio-off': audioOff,
   ban,
+  'bell-mute': bellMute,
```
Comment (codepretty) [inline packages/react/src/themes/teams/components/Icon/svg/icons/index.ts]: I recommend naming this `bell-slash`

File: packages/react/src/themes/teams/components/Icon/svg/icons/index.ts [hunk 2]
```diff
@@ -280,6 +283,7 @@ export default {
   send,
   settings,
   'share-alt': shareAlt,
+  snooze,
```
Comment (codepretty) [inline packages/react/src/themes/teams/components/Icon/svg/icons/index.ts]: This should be named `bell-snooze`

**Review by codepretty — APPROVED** [review]
Looks good - please just update the names of the icons before checking in! :)

File: packages/react/src/themes/teams/components/Icon/svg/ProcessedIcons/icons-snooze.tsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
...113.293.508.508 0 01-.137.359.458.458 0 01-.355.148.385.385 0 01-.094-.011.682.682 0 01-.086-.028 6.969 6.969 0 00-.652-.168 3.375 3.375 0 00-.676-.066 3.876 3.876 0 00-1.547.312 4.03 4.03 0 00-2.136 2.13 3.881 3.881 0 00-.317 1.558v4a1.91 1.91 0 01-.16.777 2.118 2.118 0 01-.43.637 1.974 1.974 0 01-.632.43 1.941 1.941 0 01-.778.156h12a1.943 1.943 0 01-.778-.156 2.035 2.035 0 01-.636-.43 2.057 2.057 0 01-.43-.637 1.956 1.956 0 01-.156-.777v-2a.507.507 0 01.5-.5.507.507 0 01.5.5v2a.993.993 0 00.078.39 1.043 1.043 0 00.215.317 1.032 1.032 0 00.316.215.98.98 0 00.39.078.972.972 0 01.391.078.987.987 0 01.317.215.977.977 0 01.215.316.96.96 0 01.078.391zm-8.5 2a1.453 1.453 0 00.117.578 1.522 1.522 0 00.805.805 1.485 1.485 0 001.156 0 1.52 1.52 0 00.805-.805A1.453 1.453 0 0017 22.25zm4.5-7.398a.4.4 0 01-.399.398h-2.203a.376.376 0 01-.285-.121.41.41 0 01-.113-.285.348.348 0 01.031-.149.699.699 0 01.086-.125l1.516-1.523h-1.235a.398.398 0 010-.797h2.203a.372.372 0 01.285.121.401.401 0 01.114.285.374.374 0 01-.117.274l-1.516 1.523H18.1a.4.4 0 01.399.399zm5-3.102a.507.507 0 01-.5.5h-3a.497.497 0 01-.352-.852l2.149-2.148H20a.5.5 0 010-1h3a.497.497 0 01.351.852l-2.148 2.148H23a.507.507 0 01.5.5z"
```
Comment (notandrew) [inline packages/react/src/themes/teams/components/Icon/svg/ProcessedIcons/icons-snooze.tsx:12]: File size can be reduced by running this source through svgomg:
https://jakearchibald.github.io/svgomg/

Comment (musingh1) [reply packages/react/src/themes/teams/components/Icon/svg/ProcessedIcons/icons-snooze.tsx:12]: As discussed offline, there is no change needed here.

**Pull request merged**